qemu/tests/cfg/rng_hotplug.cfg

```diff
@@ -4,18 +4,33 @@
     monitor_type = qmp
     monitors = qmp1
     repeat_times = 1
-    rng_num = 4
-    sub_test_after_hotplug = rng_bat
+    rng_num = 1
     read_rng_timeout = 360
+    no no_virtio_rng
     Windows:
         session_cmd_timeout = 240
         read_rng_cmd  = "X:\random_%PROCESSOR_ARCHITECTURE%.exe"
         driver_name = "viorng"
         rng_data_rex = "0x\w"
+        driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
```

Comment:
It should be write as below.
driver_id_cmd = X:\devcon\wxp_x86\devcon.exe find * | find "VirtIO"
driver_check_cmd = X:\devcon\wxp_x86\devcon.exe status @DRIVER_ID